conans/client/cache/editable.py

```diff
@@ -21,6 +21,9 @@ def __init__(self, cache_folder):
         else:
             self._edited_refs = {}  # {ref: {"path": path, "layout": layout}}
 
+    def refs(self):
```

Comment:
For naming conventions: this is a dictionary containing `refs` as keys, but not a list of `refs`... but we can postpone this renaming to a future PR dedicated to it.